index.qmd

```diff
@@ -53,127 +53,135 @@ bibliography: references.bib
+Constructing feasible networks and facilitating the interpolation or gap-filling of existing empirical datasets aligns with evolutionary compatibility and co-occurrence constraints. Methods embedding these scales and assumptions have been critical to the construction of 'first draft' networks for communities for which we have no interaction data [@strydomFoodWebReconstruction2022]. They are central to interpolation in data poor regions and predicting interactions for 'unobservable' communities *e.g.,* prehistoric networks [@yeakelCollapseEcologicalNetwork2014; @frickeCollapseTerrestrialMammal2022; @dunhillExtinctionCascadesCommunity2024] or future, novel community assemblages [@vanderputtenPredictingSpeciesDistribution2010]. They have the capacity to evaluate a role of interactions among species relative to their distribution by accounting for the role of the environment and the role of species interactions [@higinoMismatchIUCNRange2023; @pollockUnderstandingCooccurrenceModelling2014; @gravelBringingEltonGrinnell2019].
 
-Data implications for these approaches are that they require good datasets from which we can infer the rules, but because they contain real world species it does make it easier to validate them...
+All methods in this inference space rest on an assumption that there are a set of 'feeding rules' that underpin interaction feasibility[@morales-castillaInferringBioticInteractions2015]. With the assumption that these rules are phylogenetically conserved [@dallarivaExploringEvolutionarySignature2016; @bramonmoraIdentifyingCommonBackbone2018], and can be specified by matching the traits between consumer and resource. They represent a class of models that work by inductive reasoning. Feeding rules can be defined in multiple ways:
 
-## Construction through deduction
+The determination of the feeding rules can be defined *a priori* based expert knowledge opinions. Typically this is done on a 'trait matching' basis. An example are the paleo food web models of @shawFrameworkReconstructingAncient2024 and @roopnarineEcologicalModellingPaleocommunity2017 that specify a series of rules for a set of traits and interactions are deemed feasible if all conditions are met. Alternatively the body size ratio between the consumer and resource is often used [*e.g.,* @gravelInferringFoodWeb2013; @rohrModelingFoodWebs2010], with the idea that consumers will only utilise a resource with a body size is less than or equal to their own. However, work from @vandewalleArthropodFoodWebs2023 seems to suggest that adding morphological traits in addition to body size ratio improves model performance.
```

Comment:
I've added the very high-level bodysize ratio approaches here but at this point I don't actually know where they should fall within the different categorisations that we have used... I can see a world in which they could fall more in the deductive space but also not really...